packages/protobuf/src/messages-schema.ts

```diff
@@ -2493,30 +2556,6 @@ export const EntropyAck = Type.Object(
     { $id: 'EntropyAck' },
 );
 
-export enum RecoveryDeviceType {
```

Comment:
did it move automagically?

note for me: test on mac

Reply:
Yes, it was renamed to `RecoveryDeviceInputMethod`